scheduler/scheduler.go

```diff
@@ -151,6 +169,10 @@ func (driver *MesosSchedulerDriver) init() error {
 	driver.messenger.Install(driver.slaveLost, &mesos.LostSlaveMessage{})
 	driver.messenger.Install(driver.frameworkMessageRcvd, &mesos.ExecutorToFrameworkMessage{})
 	driver.messenger.Install(driver.frameworkErrorRcvd, &mesos.FrameworkErrorMessage{})
+
+	// install default authentication providers; extensions of this scheduler implementation
+	// are responsible for registering any additional providers they want to support.
+	auth.RegisterAuthenticateeProvider(saslProvider, auth.AuthenticateeFunc(sasl.Authenticatee))
```

Comment:
move this into auth/sasl/plugin.go::init(), and then just import the auth/sasl package anonymously here so that it registers itself